src/main/kotlin/com/team4099/robot2023/commands/Intake/SetAngleCommand.kt

```diff
@@ -0,0 +1,12 @@
+package com.team4099.robot2023.commands.Intake
+ import com.ctre.phoenix.motorcontrol.ControlMode
+import com.ctre.phoenix.motorcontrol.DemandType
+import com.ctre.phoenix.motorcontrol.NeutralMode
+import com.team4099.lib.units.derived.ElectricalPotential
+import com.team4099.lib.units.derived.inVolts
+import com.team4099.lib.units.derived.volts
+import com.team4099.robot2023.config.constants.Constants
+import com.team4099.robot2023.config.constants.Constants.Universal.CANIVORE_NAME
+import com.team4099.robot2023.config.constants.IntakeConstants
+class SetAngleCommand {
```

Comment:
oh i mistyped, i meant write setangle method not an actual command. we still need to make other commands though.